src/azure-cli/azure/cli/command_modules/appservice/static_sites.py

```diff
@@ -19,21 +19,48 @@
 logger = get_logger(__name__)
 
 
-def list_staticsites(cmd, resource_group_name=None):
+# remove irrelevant attributes from staticsites printed to the user
+def _format_staticsite(site, format_site=False):
+    if format_site:
```

Comment:
Remove these from create response only, not the Show & list - so users have a way to retrieve all the properties using CLI, if needed.

Reply:
@simonaco originally requested this because the cli "exposes some of our internal architecture that we wouldn’t want customers to interact with" -- so presumably this could apply to list and show as well. 

@simonaco do you have thoughts on this? Were you wanting us to format the output for the show/list command or just the create one?

Reply:
@StrawnSC lets remove this for create only - this will be considered breaking change for CLI - & as CLI owners we should keep these sorta breaking changes to a minimum. Ideally API should remove these properties if these shouldn't be presented to users. Since CLI is not the only client that would surface this. Thanks